### DISCUSSION TITLE
ModuleNotFoundError: No module named 'fets'

### DISCUSSION THREAD

**Discussion by ataML** [FETS-AI/Challenge#55] [Q&A] [answered]
I get the error while running the notebook. Is the module supposed to be provided or should I write it myself? 
![Screen Shot 2021-06-28 at 1 59 24 PM](https://user-images.githubusercontent.com/8337949/123633354-624e1b00-d819-11eb-827c-597322c088bc.png)

**Comment by sarthakpati** [comment, marked as answer]
Hi,

Thank you for your interest in the FeTS Challenge!

Have you installed the notebook requirements as shown in https://github.com/FETS-AI/Challenge/tree/main/Task_1#instructions?

Cheers,
Sarthak

**Reply by ataML** [reply]
Thanks, the issue was in requirements installation
